spec/spec_helper.rb

```diff
@@ -16,11 +17,17 @@
 require 'rspec/rails'
 require 'rspec/autorun'
 require 'capybara/rspec'
+require 'vcr'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.